modules/billing/README.md

```diff
@@ -0,0 +1,391 @@
+[<img src="https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg" width="400"/>](https://mineiros.io/?ref=terraform-google-billing-budget)
```

Comment:
1. Where does the README content comes from? I see references to a company called "mineiros" here...
2. Maybe this README's content can be removed or simplified?